README.md

```diff
@@ -1,6 +1,7 @@
 # PHP Akeneo PIM API
 
-A simple PHP client to use the [Akeneo PIM API](https://api.akeneo.com/).
+A simple PHP client to use the [Akeneo PIM API](https://api.akeneo.com/). 
+It is compatible with the 2.0 of our dear Akeneo PIM.
```

Comment:
No sentence on 1.7? compatible, but not for all endpoints?

Reply:
We don't support 1.7 officially. So it shouldn't be mentioned here.